modules/scripts/startup-script/files/install_ansible.sh

```diff
@@ -196,7 +196,17 @@ main() {
 		${venv_python_path} -m pip install --upgrade pip
 	fi
 
-	${venv_python_path} -m pip install -U wheel==${REQ_PIP_WHEEL_VERSION} setuptools==${REQ_PIP_SETUPTOOLS_VERSION}
+	# upgrade wheel if necessary
+	wheel_pkg=$(${venv_python_path} -m pip list --format=freeze | grep "^wheel")
+	if [ "$wheel_pkg" != "wheel==${REQ_PIP_WHEEL_VERSION}" ]; then
```

Comment:
Will this trigger the issue it's trying to solve if the current version is greater than the required version?

Reply:
Yes, it will force a downgrade. But I think that is intended behavior. Our goal (for better/worse) is to have a well defined set of versions in the ghpc-venv across all operating systems. So if a user upgrades the version of setuptools/wheel inside the ghpc-venv, they should expect it to revert upon the next execution of startup-script.